sklearn/linear_model/ridge.py

```diff
@@ -526,6 +526,8 @@ class RidgeClassifier(LinearClassifierMixin, _BaseRidge):
 
     normalize : boolean, optional, default False
         If True, the regressors X will be normalized before regression.
+        Note that regressors are always centered no matter what and no
```

Comment:
Yeah I'm no fan of that sentence either. Also, I think you should be more explicit about what "normalization" and "centering" mean.